src/video_core/renderer_vulkan/liverpool_to_vk.cpp

```diff
@@ -602,6 +605,21 @@ vk::Format AdjustColorBufferFormat(vk::Format base_format,
         default:
             break;
         }
+    } else if (comp_swap_reverse) {
+        switch (base_format) {
+        case vk::Format::eR8G8B8A8Unorm:
+            return vk::Format::eA8B8G8R8UnormPack32;
+        case vk::Format::eB8G8R8A8Unorm:
+            return vk::Format::eA8B8G8R8UnormPack32;
```

Comment:
This should be A8R8G8A8. There is no available format so should be removed from switch

Reply:
Okay that confirms what i was thinking, i will do it after my holiday trip